app/controllers/idv/image_uploads_controller.rb

```diff
@@ -80,9 +94,20 @@ def add_costs(client_response)
         call(client_response)
     end
 
-    def presenter_response(form_response, client_response)
-      return client_response if form_response.success? && client_response.present?
-      form_response
+    def presenter_response(image_form_response, client_response, doc_pii_form_response)
```

Comment:
keyword arguments pt 2
```suggestion
    def presenter_response(image_form_response:, client_response:, doc_pii_form_response:)
```